sklearn/decomposition/pca.py

```diff
@@ -488,7 +488,8 @@ class RandomizedPCA(BaseEstimator, TransformerMixin):
         use fit_transform(X) instead.
 
     iterated_power : int, optional
-        Number of iterations for the power method. 3 by default.
+        Number of iterations for the power method. 2 by default.
+        .. versionchanged:: 0.18
```

Comment:
I think what @amueller @ogrisel  suggested was to add before the parameter. You could check.